sdk/dotnet/Pulumi/Deployment/Deployment.cs

```diff
@@ -82,6 +85,9 @@ internal static IDeploymentInternal InternalInstance
         private readonly bool _isDryRun;
         private readonly ConcurrentDictionary<string, bool> _featureSupport = new ConcurrentDictionary<string, bool>();
 
+#nullable disable // remove when MemberNotNull available
+        private Serilog.ILogger _serilogger;
+#nullable restore
```

Comment:
Maybe this could be simplified to, if the idea is to workaround "Non-nullable field '_serilogger' must contain a non-null value when exiting constructor"

```suggestion
        private Serilog.ILogger _serilogger = null!;
```

Reply:
Good idea; that worked.